__tests__/api.test.js

```diff
@@ -155,6 +155,7 @@ describe("/api/articles/:article_id", () => {
 					votes: 100,
 					article_img_url:
 						"https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+					comment_count: expect.any(Number),
```

Comment:
Please try not to change old tests. This should really be a whole new test, which ideally won't break your existing test